terraform/loadgen/00_loadgen.tf

```diff
@@ -0,0 +1,150 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Let's create the GKE cluster! This one's pretty complicated so buckle up.
+
+# This is another example of the random provider. Here we're using it to pick a
+# zone in us-west1 at random - a different region from where we provisioned
+# the Hipster Shop.
+resource "random_shuffle" "zone" {
+  input = ["us-west1-a", "us-west1-b", "us-west1-c"]
```

Comment:
add other locations that are not us-west and are not locations for creation of the demo app cluster

Reply:
afaik the demo app cluster only uses `us-central1`, and loadgen cluster only uses `us-west1`... is it necessary to have a lot of possible regions? as long as they're in different geographic areas?

Reply:
yeah, different geo-locations is important. Having it be based not only in US is a secondary but might benefit folks too

Reply:
yeah I could do that! wasn't sure if it'd be restricted or something, but i can add a few other regions

Reply:
just added some, our loadgenerator is Mr. Worldwide